### ISSUE TITLE
Bullet is not properly aligned

### ISSUE THREAD

**Issue by tim-hilde** [kepano/obsidian-minimal#92]
The bullet in LP is not properly aligned

<img width="374" alt="Bildschirmfoto 2021-12-31 um 00 01 02" src="https://user-images.githubusercontent.com/44113468/147793301-9dda1e8e-c665-4617-a1ab-93f2e4713822.png">

**Comment by kepano** [comment]
Which version of Obsidian and Minimal Theme are you using?

Obsidian 0.13.17 has several changes for bullets in LP.

**Comment by tim-hilde** [comment]
I'm using 0.13.17 and the most recent version of Minimal Theme

**Comment by kepano** [comment]
Can you try updating to Minimal 4.3.2 and let me know how that looks?

**Comment by tim-hilde** [comment]
Thank you for the fast reply. Updating doesn't change anything...

**Comment by tim-hilde** [comment]
Okay, maybe this helps: it definitely has to do with the iA Quattro font I'm using. Changing it solves the styling of the bullet point (circle instead of square), as well as the alignment.

**Comment by kepano** [comment]
I am trying it with iA Quattro but it doesn't look as bad on my end.

Obsidian team are currently making changes to bullets and gutter behavior in Live Preview with each new version so I am going to wait for things to stabilize before I continue to spend more time tweaking this.

In the meantime you can use the following snippet and adjust the line height value to match your preferences

```
.mod-cm6 .HyperMD-list-line .list-bullet::after,
.mod-cm6 span.list-bullet::after {
line-height:1em;
}
```

**Comment by queckezz** [comment]
@kepano I only updated to the latest Minimal Theme Settings but forgot to update the theme itself. That fixed it for me.

![image](https://user-images.githubusercontent.com/2580598/148531297-b1f3e75a-1ed3-41ef-9488-6afb9f60a22a.png)

**Comment by kepano** [comment]
I believe this is now fixed as of 4.2.2
